components/BottomNavigation/README.md

```diff
@@ -181,3 +186,65 @@ id<MDCTypographyScheming> typographyScheme = [[MDCTypographyScheme alloc] init];
 ```
 <!--</div>-->
 
+
+## Accessibility
```

Comment:
Was this auto-generated? The source file seems to have a different header than this.